test/common/vmimages.py

```diff
@@ -38,7 +38,11 @@
 DEVNULL = open("/dev/null", "r+")
 
 CONFIG = "~/.config/image-stores"
-DEFAULT = "https://fedorapeople.org/groups/cockpit/images/"
+DEFAULT = [
+    "https://209.132.184.69:8493/",
```

Comment:
This is fine for now of course (as this is urgent), but how reliable are these hardcoded IPs? This will eventually land in stable releases, branches, etc.